runtime/lua/vim/fs.lua

```diff
@@ -732,4 +732,28 @@ function M.abspath(path)
   return M.joinpath(cwd, path)
 end
 
+--- Return relative path from directory `base` to `target`. Note that `base` must be an ancestor to
+--- `target`, meaning relative paths starting with `../` are not supported.
```

Comment:
Not clear what `paths starting with ../ are not supported` means. `abspath()` would handle such inputs.